dali/pipeline/util/copy_with_stride_test.cc

```diff
@@ -78,11 +79,10 @@ TEST(CopyWithStrideTest, TwoDimsGPU) {
                      41, 42, 43, 44};
   Index stride[] = {8 * sizeof(size_t), sizeof(size_t)};
   Index shape[] = {2, 4};
-  size_t *data;
-  CUDA_CALL(cudaMalloc(reinterpret_cast<void**>(&data), sizeof(size_t) * 16));
+  DeviceBuffer<size_t> data, out;
+  data.from_host(h_data);
```

Comment:
Do you need `cudaMemcpy` if you call `from_host`?

Reply:
No, I don't.